_src/static/sass/main-ltr.scss

```diff
@@ -13,17 +13,21 @@
 @import '../../vendor/bi-app/bi-app-ltr';   // LTR support
 
 // ------------------------------
-// #UTILITIES
```

Comment:
We definitely need `_variables.scss` above things that are using/referencing them, but it looks like in the styleguide compile example, `_variables.scss` is placed in utilities as `sass/utilities/_variables.scss`. Mind moving it to where we planned? This should allow for the use later as well.